frontend/src/components/pages/NotAuthorized.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const NotAuthorized = (): React.ReactElement => {
+  return (
+    <div style={{ textAlign: "center" }}>
+      <h1>Can&apost view this page since you don&apost have permission 🙁</h1>
```

Comment:
Canada post? 📬
![image](https://github.com/uwblueprint/extend-a-family/assets/82423082/7e7c5df7-a55d-4380-b97d-16f86465fab1)
you're probably good to just use a normal keyboard apostrophe ' , unless there's a special reason for using an escaped apostrophe? :D

Reply:
eslint doesn't like using normal keyboard apostrophe. should i just ignore it?

Reply:
OHH okay, let's use the escaped apostrophes then - `Can&apos;t` and `don&apos;t` (you were missing the semicolon to end the escaped string lol)